BlueprintUICommonControls/Sources/AttributedLabel.swift

```diff
@@ -514,6 +523,27 @@ extension AttributedLabel {
                         link: link
                     )
                 }
+
+            // Insert the word "link" after each link in the label. This mirrors the VoiceOver utterance of `SwiftUI.Text` when rendering a markdown link.
+            guard var label = accessibilityLabel,
+                  let localizedLinkString = linkAccessibilityLabel,
+                  !links.isEmpty else { return }
+
+            let insertionString = "[\(localizedLinkString)] "
```

Comment:
Do the `[ ]` have any special meaning here?

Reply:
its really only for the case where the string would be printed. e.g. the accessibility inspector. I wanted to be clear that it's not a part of the displayed text. and that it's explicitly associated with the link string (it doesn't have a preceding space). 

I chose the square brackets because
- Voiceover doesn't read the `[]` characters, but does realize the contained word is distinct from the preceding word. 
- They aren't often used in prose, unlike parenthesis. They're unlikely to be confused with the actual content
- They look like markdown.

Reply:
Makes sense! It might be good to capture that in a comment inline